src/main/webapp/creatives/maps/simid_maps.js

```diff
@@ -1,8 +1,44 @@
 /**
  * A sample SIMID ad that shows a map of nearby locations
  */
+const AdParamKeys = {
+  BUTTON_LABEL: 'buttonLabel',
+  SEARCH_QUERY: 'searchQuery',
+  MARKER: 'marker',
+}
+const DEFAULT_BUTTON_LABEL = "Location";
+
 class SimidMapCreative extends BaseSimidCreative {
   constructor() {
     super();
   }
+
+  /** @override */
+  onStart(eventData){
+    super.onStart(eventData);
+    const adParams = this.creativeData.adParameters;
+    const buttonLabel = JSON.parse(adParams)[AdParamKeys.BUTTON_LABEL]; 
+    if (buttonLabel == undefined) {
+      buttonLabel = DEFAULT_BUTTON_LABEL;
+    }
+    else {
+      buttonLabel = JSON.parse(adParams)[AdParamKeys.BUTTON_LABEL];
+    }
+    let searchQuery = JSON.parse(adParams)[AdParamKeys.SEARCH_QUERY];
```

Comment:
searchQuery isn't being reassigned, so it can be `const`

Even if you pass it to another function, it can be const.  When you call the next function, it will pass the variable by value, so you only need let if the variable gets reassigned within the same function

Reply:
done